packages/components/addon/components/hds/form/field/index.js

```diff
@@ -10,6 +10,8 @@ export const LAYOUT_TYPES = ['vertical', 'flag'];
 export default class HdsFormFieldIndexComponent extends Component {
   @tracked ariaDescribedBy = getAriaDescribedBy(this);
   @tracked descriptors = [];
+  @tracked isRequired = this.args.isRequired;
+  @tracked isOptional = this.args.isOptional;
```

Comment:
We don't need a setter on these values, but we at least need a getter, because of [the way we forward them to the label/legend component](https://github.com/hashicorp/design-system/pull/425/files#diff-61b99252eb7c03439d9402878520f0bfe18b8ecdf6e9d5c71188cdc0035ef168R9).

I can convert them from `@tracked` to only having a getter, e.g.
```
  get isRequired() {
    let { isRequired } = this.args;
    return isRequired;
  }
```

Reply:
Mmm, ok now I see why (that was the part I missed: the fact that when you hash them, they need to be `this.something` and not `this.args.something`, while I was thinking "in React" where automatically a prop is automatically exposed)

Personally, I have a subtle preference for the getter, probably in an ever simplified way:
```
// I would add a comment here explaining why we need to do this strange thing
get isRequired() {
    return this.args.isRequired;
  }
```